### ISSUE TITLE
Duplicated categories

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#77]
```
What's the problem?
when i get the categories in my blog, every category is duplicated. i have 6 
categories in my blog, and it shows me 12, 2 for each category.




What version of ScribeFire for Chrome are you running?
1.0.0.1
```

-----
Original issue reported on code.google.com by `brianden...@gmail.com` on 25 Jun 2010 at 5:07

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `cfinke` on 24 Jul 2010 at 2:37
* Changed state: **Accepted**
* Added labels: **Milestone-1.2**

**Comment by GoogleCodeExporter** [comment]
```
Fixed at some point before 1.1.0.0.
```

-----
Original comment by `cfinke` on 24 Jul 2010 at 2:56
* Changed state: **Fixed**
* Removed labels: **Milestone-1.2**

**Comment by GoogleCodeExporter** [comment]
```
I see the changes, thanks.
```

-----
Original comment by `brianden...@gmail.com` on 24 Jul 2010 at 11:33
